apidef/oas/event.go

```diff
@@ -10,9 +10,11 @@ import (
 )
 
 // Kind is an alias maintained to be used in imports.
+// nolint
```

Comment:
This is probablly wrong,...

Reply:
Remove `// nolint` hints, if the intent is to skip some symbols, we need to adjust the CI action.
We already skip a few symbols from getting exported to docs here:
https://github.com/TykTechnologies/exp/blob/main/.github/workflows/tyk-docs.yml#L154